.github/workflows/release-project.yaml

```
@@ -31,64 +31,29 @@ env:
 
 jobs:
   release_project:
-    name: Release Husarion UGV project
+    name: Release project
     runs-on: ubuntu-22.04
     steps:
-      - name: Release panther_ros repository
-        uses: convictional/trigger-workflow-and-wait@v1.6.1
+      - name: Release repository
+        uses: ./.github/workflows/release-repository.yaml
         with:
-          owner: husarion
-          repo: panther_ros
-          github_token: ${{ secrets.GITHUB_TOKEN }} # Use the default GITHUB_TOKEN for local repository
-          workflow_file_name: release-repository.yaml
-          ref: ${{ env.RC_BRANCH_NAME }}
-          client_payload: |
-            {
-              "release_candidate": "${{ env.RC_BRANCH_NAME }}",
-              "version": "${{ github.event.inputs.version }}",
-              "release_name": "${{ github.event.inputs.release_name }}",
-              "automatic_mode": "${{ github.event.inputs.automatic_mode }}",
-              "prerelease": "${{ github.event.inputs.prerelease }}"
-            }
+          release_candidate: ${{ env.RC_BRANCH_NAME }}
+          version: ${{ github.event.inputs.version }}
+          release_name: ${{ github.event.inputs.release_name }}
+          automatic_mode: ${{ github.event.inputs.automatic_mode }}
+          prerelease: ${{ github.event.inputs.prerelease }}
 
-      - name: Rebuild panther docker images with new version
-        uses: convictional/trigger-workflow-and-wait@v1.6.1
+      - name: Build development Docker
```

```suggestion
      - name: Rebuild panther docker images with new version
```
Old name was more descriptive. This is required to run after repository is released and version in package.xml was updated. This is not development image, it is release image. We pass build_type: development so the image tag does not have `-stable` at the end

---

Steps are executed one by one, there is no need to create sepret jobs with `need` field for that.